Vindicta.Altis/Templates/initFactions.sqf

```diff
@@ -6,27 +6,28 @@
 // !! Factions will be listed in UI in the same order as here !! 
 
 // Military factions
```

Comment:
All this indentation is messed up, are you using 2 space tabs instead of 4?

Reply:
I'm using 4 space tabs